src/main/host/syscall/handler/unistd.rs

```diff
@@ -933,10 +933,11 @@ impl SyscallHandler {
 
         let nodename = u8_to_i8_slice(ctx.objs.host.info().name.as_bytes());
 
-        let sysname = u8_to_i8_slice(&b"shadowsys"[..]);
-        let release = u8_to_i8_slice(&b"shadowrelease"[..]);
-        let version = u8_to_i8_slice(&b"shadowversion"[..]);
-        let machine = u8_to_i8_slice(&b"shadowmachine"[..]);
+        // Currently hardcoded with values reported in Debian 12
+        let sysname = u8_to_i8_slice(&b"Linux"[..]);
+        let release = u8_to_i8_slice(&b"6.1.0-25-amd64"[..]);
+        let version = u8_to_i8_slice(&b"#1 SMP PREEMPT_DYNAMIC Debian 6.1.106-3 (2024-08-26)"[..]);
```

Comment:
`SMP` and `PREEMPT_DYNAMIC` seem a little unfortunate since we emulate a single-core machine with no preemption. Unless we have an example of a "real" string without those I suppose it's probably better to just leave them though.

Reply:
This might be a configure option when the kernel is built. It looks like we could at least just drop the `PREEMPT_DYNAMIC` part

https://www.reddit.com/r/linuxquestions/comments/sc14th/change_kernel_version_during_build_uname_v/

and we could probably dig for more examples.

Reply:
Yeah, IDK. Dropping `PREEMPT_DYNAMIC` seems more truthful, but also as likely to cause problems with code trying to do sniffing here than solve them since it's probably "weird".

I'm fine with either way. I think I'd lean towards just keeping what you have, and if we run into more related problems making it configurable.